ush/python_utils/environment.py

```diff
@@ -32,31 +32,30 @@ def str_to_date(s):
 
 
 def date_to_str(d, format="%Y%m%d%H%M"):
-    """Get string from python datetime object.
-    By default it converts to YYYYMMDDHHMM format unless
-    told otherwise by passing a different format
+    """Gets string from Python datetime object.
+    By default it converts to ``YYYYMMDDHHmm`` format unless told otherwise by passing a different format.
 
     Args:
-        d: datetime object
+        d (datetime.datetime): datetime object
+        format (str): Format of the datetime string; default is ``"%Y%m%d%H%M"`` (see `format codes <https://docs.python.org/3/library/datetime.html#format-codes>`_ for other options).
     Returns:
-        string in YYYYMMDDHHMM or shorter version of it
+        String in YYYYMMDDHHmm or shorter version of it
     """
     v = d.strftime(format)
     return v
 
 
 def str_to_type(s, return_string=0):
-    """Check if the string contains a float, int, boolean, datetime, or just regular string.
+    """Checks whether the string is a float, int, boolean, datetime, or just regular string.
     This will be used to automatically convert environment variables to data types
     that are more convenient to work with. If you don't want this functionality,
-    pass return_string = 1
+    pass ``return_string = 1``.
 
     Args:
-        s: a string
-        return_string: Set to 1 to return the string itself
-                       Set to 2 to return the string itself only for a datetime object
+        s (str): a string
```

Comment:
Should the a be capitalized?

```suggestion
        s (str): A string
```